src/inmanta/moduletool.py

```diff
@@ -103,17 +103,21 @@ def add_deps_check_arguments(parser: argparse.ArgumentParser) -> None:
         dest="no_strict_deps_check",
         action="store_true",
         default=False,
-        help="When this option is enabled, only version conflicts in the direct dependencies will result in an error. "
-             "All other version conflicts will result in a warning. This option is mutually exclusive with the "
+        help=(
+            "When this option is enabled, only version conflicts in the direct dependencies will result in an error. ",
+             "All other version conflicts will result in a warning. This option is mutually exclusive with the ",
              "--strict-deps-check option.",
+        ),
     )
     parser.add_argument(
         "--strict-deps-check",
         dest="strict_deps_check",
         action="store_true",
         default=False,
-        help="When this option is enabled, a version conflict in any (transitive) dependency will results in an error. "
+        help=(
+            "When this option is enabled, a version conflict in any (transitive) dependency will results in an error. ",
              "This option is mutually exclusive with the --no-strict-deps-check option.",
```

Comment:
```suggestion
            "This option is mutually exclusive with the --no-strict-deps-check option.",
```

Reply:
I have reunified the string. Otherwise, the test_feature_flags test was failing because string methods were applied to a tuple.

Reply:
Ah, I missed the comma. I don't think the way you made it now is good for this use case. It is a multi-line string, meaning it includes all newlines and indentation. We don't want that. We want it to be a single, one-line string, but we don't necessarily want to write it in our source as such. That's where the string concatenation comes in. The only issue with the above is the comma's at the end of the line:

```python
>>> help = (
...     "hello",
...     " world",
... )
>>> help
('hello', ' world')
>>> help = (
...     "hello"
...     " world"
... )
>>> help
'hello world'
>>> 
```

So there should only be a comma after the closing bracket.